src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,33 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int EMPTY_FIFO = -1;
+    private static final int FULL_FIFO = 11;
```

Comment:
This constant is related to CAPACITY . Changing only one leads to an error (inconsistency).